node/flows.re

```diff
@@ -419,6 +419,11 @@ let trusted_validators_membership =
   let {signature, payload: {address, action} as payload} = request;
   let payload_hash =
     payload |> payload_to_yojson |> Yojson.Safe.to_string |> BLAKE2B.hash;
+  let.assert () = (
+    `Invalid_signature_author,
+    Address.compare(state.Node.identity.t, Signature.public_key(signature))
+    == 0,
```

Comment:
Why not Int.equal? Polymorphic comparison is quite bad.(https://blog.janestreet.com/the-perils-of-polymorphic-compare)

Reply:
Polymorphic comparison is mostly a problem for structured data structures, but I also agree that `==` should be Int.equal like on Base. #217